### PR TITLE
Add espn

### PR DESCRIPTION
**Pull request by Theitfixer85**
Add espn to us m3u

### CODE REVIEW

**Review by freearhey — CHANGES_REQUESTED** [review]
Please use tag `#EXTVLCOPT:http-referrer` to specify a referrer. 
More info here: https://github.com/iptv-org/iptv/blob/master/.github/CONTRIBUTING.md#add-channel

**Comment by Theitfixer85** [comment]
@freearhey patch updated with correct format

File: channels/us.m3u [hunk 1]
```diff
@@ -305,6 +305,9 @@ https://etlive-mediapackage-fastly.cbsaavideo.com/dvr/manifest.m3u8
 https://stream.swagit.com/live-edge/escambiacountyfl/live-1-a/playlist.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="English" tvg-logo="https://i.imgur.com/fGdHIGy.png" group-title="Religious",ESNE TV
 https://zypelive-lh.akamaihd.net/i/default_1@710948/master.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="English" tvg-logo="https://static.epg.best/us/ESPN.us.png" group-title="Sport",ESPN
+#EXTVLCOPT:http-referrer=https://espn-live.stream/stream/43572.html
+https://e10.cdn4.us/ingest666/espn_usa/f.m3u8
```
Comment (freearhey) [inline channels/us.m3u:310]: The stream's not working at the moment. Tested in VLC player.
Screenshot 2020-07-27 at 17 36 42

**Review by freearhey — CHANGES_REQUESTED** [review]
Please check marked link again.

File: channels/us.m3u (hunk 1 above)
Comment (Theitfixer85) [reply channels/us.m3u:310]: Yes was working all weekend but not working now - I will close the pull until I find a working link again

**Comment by Theitfixer85** [comment]
Can you check again - it's working for me now

**Comment by Theitfixer85** [comment]
Hmm it works in the original format I posted but not in the amended format you wanted so I'll have to close for now and test more

**Pull request closed without merging**